pandas/tests/groupby/test_groupby.py

```diff
@@ -3159,3 +3159,29 @@ def test_groupby_series_with_datetimeindex_month_name():
     expected = Series([2, 1], name="jan")
     expected.index.name = "jan"
     tm.assert_series_equal(result, expected)
+
+
+def test_rolling_corr_grouping_column_contains_tuples_1(df):
```

Comment:
Could you move these tests to `pandas/tests/window/test_groupby.py` in a follow up PR?

Reply:
Yeah sounds good, will get it in shortly

Reply:
Hey @mroeschke, this is the number for the new pr #54504, I'm just waiting for the checks to clear.

Reply:
@mroeschke the other PR went through please merge :) thanks